### ISSUE TITLE
goodixgt9110P not working on iota flo360

### ISSUE THREAD

**Issue by gingerjedi22** [sebanc/brunch#1581]
Hi, using 101 rammus, with kernel 5.4, have enabled goodix, and tried enabling touch with the search+T command, to no avail.

I have managed to get bluetooth and wifi working.

Can anyone guide me as to what I am doing wrong.

Screens working fine on W11.

Kind Regards.

**Comment by gingerjedi22** [comment]
i have enable goodix on the 5.4, 5.10. and 5.15, and no joy.

**Comment by tejasraman** [comment]
Try `modprobe goodix` in a shell (go to crosh and type "shell" and inside shell run that command) - to get to crosh, Ctrl+Alt+T

**Comment by gingerjedi22** [comment]
Thanks tejasraman, my skill level here is very weak but i will have a go, your help is so much appreciated, kind regards.

**Comment by tejasraman** [comment]
That is to see if the goodix kernel module (basically, a driver) is being used. Oh, I made a mistake. Run this instead; `sudo modprobe goodix`

**Comment by tejasraman** [comment]
I'll send a video if needed

**Comment by gingerjedi22** [comment]
going to attempt a fix tonight, im sure when i compiled I had entered the right commands, I got the wifi and bluetooth modules working, and selected the goodix option, hopefully i can find the right commands to enable it or recompile another build, this is for a midi touchscreen sampler app called remixlive, i have 3 control surfaces for midi, and it would be useful to employ the touchscreen effects section the app has

**Comment by gingerjedi22** [comment]
so, i open cosh, type in shell then the sudo command line ?

**Comment by tejasraman** [comment]
Yes, and type `sudo modprobe goodix` there and see if it works. If so, we can create a boot override so that it works without that command

**Comment by gingerjedi22** [comment]
mmm, nothing

**Comment by tejasraman** [comment]
Run it and take a pic please

**Comment by gingerjedi22** [comment]
no, sorry, no response

**Comment by tejasraman** [comment]
Try `lsmod | grep goodix`
Also, try disabling goodix in brunch config and try modprobe again

**Comment by tejasraman** [comment]
Tell me the output of the lsmod command first.....

**Comment by gingerjedi22** [comment]
command not found, not sure iv got the command right

**Comment by tejasraman** [comment]
It is L S M O D | G R E P  G O O D I X
`lsmod | grep goodix`

**Comment by gingerjedi22** [comment]
![PXL_20220714_154413940](https://user-images.githubusercontent.com/107829829/179022966-2ea703ba-f5ae-4193-95e7-3d21fe317d4e.jpg)

**Comment by gingerjedi22** [comment]
lo, i got there lol

**Comment by tejasraman** [comment]
Weird... the goodix driver is loaded on your system.
Try disabling enable_goodix in the brunch config and then running the `modprobe goodix` command again

**Comment by gingerjedi22** [comment]
trying without goodix enabled, 5.1.

**Comment by tejasraman** [comment]
Do 5.15 and no goodix
And then do the modprobe to load goodix manually

**Comment by gingerjedi22** [comment]
k

**Comment by tejasraman** [comment]
Thanks. Report back after running modprobe

**Comment by gingerjedi22** [comment]
5.1 modprobe not working
trying 5.15

**Comment by tejasraman** [comment]
Remember, modprobe will be blank. Please try the screen after modprobe

**Comment by tejasraman** [comment]
Kernel 5.1? What version of brunch are you on?????

**Comment by gingerjedi22** [comment]
it was blank, but no touchscreen response
and my wifi failed to load, trying again

**Comment by gingerjedi22** [comment]
5.15 was a no go, trying 5.4

**Comment by tejasraman** [comment]
Try to modprobe after every one please

**Comment by gingerjedi22** [comment]
going back to 5.10, the wfi, bluetooth, and andriod launcher all work, will remove goodix and try modprobe

**Comment by gingerjedi22** [comment]
![PXL_20220714_162937205 MP](https://user-images.githubusercontent.com/107829829/179031838-caff9ceb-364d-4dfd-8194-172c219dcdd9.jpg)
5.10 goodix not selected

**Comment by gingerjedi22** [comment]
no touch

**Comment by gingerjedi22** [comment]
im gonna check my windows install, just in case there has been a bios change

**Comment by tejasraman** [comment]
Ok..... try chrome os flex and report back pls.

**Comment by tejasraman** [comment]
Chrome os flex?

**Comment by gingerjedi22** [comment]
no mate i will purchase a tablet or chromebook, that easily solves my problem
________________________________

**Comment by tejasraman** [comment]
Ok
